ChrisPerez/app/js/first/directives/index.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function(app) {
+  // require('./FirstDirective')(app);
```

Comment:
if these aren't being used - good idea to remove from project - could cause problems